### ISSUE TITLE
Pre-Processing: Training and Validation data

### ISSUE THREAD

**Issue by perseus93** [PythEsc/Research_project2#14]
We want to create a training and validation set.

We agreed to pick random posts over time since the topics might change.
Training Set: 90% of data
Validation Set: 10% of data
